src/app/solution-center/os-install/os-install.component.scss

```diff
@@ -137,8 +123,41 @@
     background: #fafafa;
   }
 
-  .input-width {
-    width: 100%;
+  .required {
+    &:after{
+      content: "*";
+      font-size: 1.1em;
+      color: #c92100;
+      margin-left: .25rem;
+    }
+  }
+
+  .label-one-line {
+    white-space: nowrap;
+    width: fit-content;
+    max-width: 100%;
+  }
+
+  .form-control{
+    flex-wrap: nowrap;
+  }
+  .fieldset .clr-input-wrapper{
+        border: red solid 1px;
+        max-width: 100%;
+        flex: 0 0 0 !important;
+      }
+
+
+  .clr-input-wrapper{
+    width: 100%
+  }
+
+  .input-width{
+    width: 100%
+  }
+
+  .float-left{
+    float: left !important;
```

Comment:
Ok, so not sure about @asantos00 but i *really* don't like css `float`, can you figure another way around that ?

Reply:
well speaked with him, he is also no super in favor of that